### PR TITLE
service.audio.profiles 1.0.0

### PR DESCRIPTION
**Pull request by Regss**

### CODE REVIEW

File: service.audio.profiles/addon.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="service.audio.profiles"
+    name="Audio Profiles Service"
+    version="1.0.0"
+    provider-name="Regss">
+    <requires>
+        <import addon="xbmc.python" version="2.1.0" />
+    </requires>
+    <extension point="xbmc.service" library="default.py" start="[login|startup]" />
```
Comment (ronie) [inline service.audio.profiles/addon.xml]: start="[login|startup]" is not valid ;-)

use either start="login" or start="startup".
see: http://kodi.wiki/view/Service_addons

**Comment by Regss** [comment]
start="[login|startup]" it worked for me :). Fixed.

**Comment by ronie** [comment]
thanx, could you squash both commits into one?

**Comment by Regss** [comment]
I will try but this is for advanced git users :)

**Comment by ronie** [comment]
if you're using git cli, it should be along these lines:
git rebase -i HEAD~2
(follow instructions on screen0
git push -f origin gotham

**Comment by kib** [comment]
I just happened to write an extended explanation here: https://github.com/kib/skin.neon/pull/68#issuecomment-70656918

might come in useful :+1:

**Comment by Regss** [comment]
Yeah! Well done. Experience +3000 Level up.

**Comment by ronie** [comment]
added, thx!

**Pull request merged**